### DISCUSSION TITLE
v2.0.1 Nuget packages

### DISCUSSION THREAD

**Discussion by daveatdnv** [DotSpatial/DotSpatial#1420] [General]
Will the v2.0.1 release be available as Nuget packages or should we switch to using the assemblies directly?

**Comment by jany-tenaj** [comment]
@danames @mogikanin Will one of you upload the v2.0.1 nuget packages to nuget?

@daveatdnv You could also use the [nuget feed](https://ci.appveyor.com/nuget/dotspatial) of the continuous integration builds.

**Reply by daveatdnv** [reply]
Thanks @jany-tenaj. We would prefer to use the Nuget feed, so it would be great if @danames or @mogikanin could upload the packages to Nuget.

**Comment by danames** [comment]
Hey all, it looks like @mogikanin has the email address associated with the dotspatial nuget account. I've forwarded him an email to see if he can get that updated or transfer the account to someone else. - Dan

**Comment by mogikanin** [comment]
Hi there, I've published v2.0.1.2 to official nuget feed. Please check and confirm it works as expected 😄

**Reply by daveatdnv** [reply]
Working as expected, thanks very much.

**Comment by sindizzy** [comment]
Wait...there's a 2.0.1.2 release? Was this made public? I didnt see any emails on this.

**Reply by daveatdnv** [reply]
There have been several releases on Github (https://github.com/DotSpatial/DotSpatial/releases) with the associated packages on Nuget (https://www.nuget.org/profiles/dotspatial).
